CHANGELOG.md

```diff
@@ -983,6 +984,16 @@ Adding a new version? You'll need three changes:
 [KongIngress to KongUpstreamPolicy migration guide]: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/migrate/kongingress/
 [Migrate Credential Type Labels]: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/migrate/credential-kongcredtype-label/
 
+## [2.12.7]
+
+> Release date: 2024-11-25
+
+### Fixed
+
+- Bump go-kong to v0.56.0 to fix [#6703](https://github.com/Kong/kubernetes-ingress-controller/issues/6703)
```

Comment:
We might be a bit more verbose about what 6703 is in here, e.g. mention what plugin issue this is solving.

Reply:
```suggestion
- Bump go-kong to v0.56.0 to fix [#6703](https://github.com/Kong/kubernetes-ingress-controller/issues/6703) 
  This way, the OTEL plugin can work properly when the Gateway is upgraded from 3.6 to 3.7.
```

@pmalek WDYT?

Reply:
Sounds good.